### ISSUE TITLE
onState not being called

### ISSUE THREAD

**Issue by BrendanNeufeld** [Jam3/f1#8]
The following function never gets called:
```
this.ui = new f1({
        onState: function(state){
          console.log('state: ',arguments);
        }
      })
```
Even calling `this.ui.onState()` doesn't execute the function. It must not be getting assigned.

**Comment by MikkoH** [comment]
@BrendanNeufeld I can look into this. I've been thinking of using events for state updates. However I was thinking that this should be backwards compatible and you can use a cb if you want.

Is this crucial to get fixed like RIGHT now?

**Comment by BrendanNeufeld** [comment]
not urgent.

**Comment by MikkoH** [comment]
This is fixed in `v5.2.3`
